layers/src/main/scala/geotrellis/layers/mask/Mask.scala

```diff
@@ -45,9 +44,12 @@ object Mask {
     def DEFAULT = Options()
     implicit def rasterizerOptionsToOptions(opt: Rasterizer.Options): Options = Options(opt)
   }
+}
 
+trait Mask {
+  import Mask._
   // As done by [[geotrellis.raster.rasterize.polygon.TestLineSet]] in [[geotrellis.raster.rasterize.polygon.PolygonRasterizer]].
-  private def eliminateNotQualified(geom: Option[Geometry]): Option[Geometry] = {
+  private[geotrellis] def eliminateNotQualified(geom: Option[Geometry]): Option[Geometry] = {
```

Comment:
Yes. This method being private to file means that it could not have an impact on binary compatibility since its only callable from code that will be recompiled and shipped with any possible changes. Scoping it to `geotrellis` raises possibility that becomes a dependency on for another `geotrellis` package which could be available with different version to `geotrellis.layers` (as result of some kind of dependency resolution by SBT).

That is still not directly "user API" and shouldn't happen often, but is a much softer guarantee.